bin/yk-config

```diff
@@ -43,20 +43,30 @@ AOTPASSES="${AOTPASSES},rpo-function-attrs,globalsplit,wholeprogramdevirt"
 #
 # FIXME: skipping some problematic passes for now:
 #
-# `mem2reg` (a.k.a. `PromotePass`), `sroa`, `jumpthreading`, `tailcallelim`,
-# `licm`, `gvn`, `simplifycfg`.
-#
 # These cause test failures and need to be investigated. When adding these
 # back, be sure to put them in the right place, and check if they should run
 # more than once in the pipeline.
-AOTPASSES="${AOTPASSES},globalopt,constmerge,instcombine"
+AOTPASSES="${AOTPASSES},globalopt"
+#AOTPASSES="${AOTPASSES},mem2reg" -- breaks Yk test suite.
+AOTPASSES="${AOTPASSES},constmerge,instcombine"
 # Note `aggressive-instcombine` is O3.
-AOTPASSES="${AOTPASSES},aggressive-instcombine,globalopt,globaldce"
-AOTPASSES="${AOTPASSES},argpromotion,deadargelim,instcombine,function-attrs"
-AOTPASSES="${AOTPASSES},memcpyopt,dse,mldst-motion,loop-flatten,indvars"
+#
+# FIXME: Also note that `inline` probably isn't doing anything since we require
+# the interpreter to be built -O0, menaning clang will slap `noinline`
+# everywhere.
+AOTPASSES="${AOTPASSES},aggressive-instcombine,inline"
+AOTPASSES="${AOTPASSES},globalopt,globaldce,argpromotion,deadargelim"
+#AOTPASSES="${AOTPASSES},instcombine" -- this second invocation breaks yklua
+#AOTPASSES="${AOTPASSES},jumpthreading,sroa,tailcallalim" -- break Yk test suite.
```

Comment:
I think `yk` is lower case?

Reply:
it can be :) Will fix.

Reply:
I don't see two new commits?

Reply:
refresh?

![image](https://github.com/ykjit/yk/assets/604955/d1de0aaa-f047-46d9-9f04-cd76ab25fdeb)